spec/api/comment_thread_spec.rb

```diff
@@ -565,7 +565,6 @@ def test_unicode_data(text)
 
       it "marks thread as read and confirms its value on returned response" do
         user = create_test_user(123)
-        thread = CommentThread.first
```

Comment:
It seems strange that we are removing this variable declaration, but still referencing it in the lines below. Can you explain this change?